python/lib/metrics/__init__.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+import typing
+
+from argparse import ArgumentParser
+
+
+class Arguments(object):
+    def __init__(self, *args):
+        self.__option: typing.Optional[str] = None
+
+        self.__parse_command_line(*args)
+
+    @property
+    def option(self) -> str:
+        return self.__option
+
+    def __parse_command_line(self, *args):
+        parser = ArgumentParser("Put a description for your script here")
+
+        # Add options
+        parser.add_argument(
+            "-o",
+            metavar="option",
+            dest="option",
+            type=str,
+            default="default",
+            help="This is an example of an option"
+        )
+
+        # Parse the list of args if one is passed instead of args passed to the script
+        if args:
+            parameters = parser.parse_args(args)
+        else:
+            parameters = parser.parse_args()
+
+        # Assign parsed parameters to member variables
+        self.__option = parameters.option
+
+
+def main():
+    """
+    Define your initial application code here
+    """
+    arguments = Arguments()
+
+
+# Run the following if the script was run directly
+if __name__ == "__main__":
```

Comment:
Definitely not major, but we may want to move the main function into `__main__.py` so it can be run using `python -m dmod.metrics`.

Reply:
That whole file is left over from an old template. I'm going to wipe the file to get all of that out of it, but it's questionable whether or not there should be an init there - it might end up conflicting with other inits when building and that's not something I'm prepared to try to break and fix.